lib/Cleantalk/ApbctWP/AdminNotices.php

```diff
@@ -192,6 +194,8 @@ public function notice_trial() // phpcs:ignore PSR1.Methods.CamelCapsMethodName.
      */
     public function notice_renew() // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
     {
+        global $apbct;
+
         if ( $this->apbct->notice_show && $this->apbct->notice_renew == 1 && $this->apbct->moderate_ip == 0 && ! $this->apbct->white_label ) {
             $renew_link  = "<a href=\"https://cleantalk.org/my/bill/recharge?utm_source=wp-backend&utm_medium=cpc&utm_campaign=WP%%20backend%%20renew$this->user_token&cp_mode=antispam\" target=\"_blank\">%s</a>";
```

Comment:
Probably we should replace this with mailto link if it was provided by wl.

Reply:
One of condition for show notice on client dashboard $this->apbct->white_label. It's mean notice already hidden in white_label mode.